components/repository-issues.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+
+import Link from "next/link"
+import React from "react"
+
+import { useGetRepositoryIssues } from "@/hooks/useGetRepositoryIssues"
+
+import Badge from "./badge"
+import Skeleton from "./skeleton"
+
+const RepositoryIssues = () => {
+    const { error, issues, loading } = useGetRepositoryIssues()
+    console.log({ loading })
```

Comment:
We can remove this console.log

Reply:
addressed -> https://github.com/bitcoin-dev-project/bitcoin-dev-project/pull/41/commits/31cfca3c7933387a8f8d2d6b1c0789d6dadb3ff6